hd/lang/lex_utf8.txt

```diff
@@ -28,6 +28,10 @@ fr: dictionnaire des %s
 en: compiled on
 fr: compilé le
 
+    commited on
```

Comment:
Compiles and commited are two separate entries.
I dont understand the comment!!